### PR TITLE
If hostname is not same as redirectUrl, redirect user

### PR DESCRIPTION
**Pull request by IvanBuljovcic**

### CODE REVIEW

File: src/types/Venue.ts [hunk 1]
```diff
@@ -105,6 +105,8 @@ export interface VenueConfig {
 
   landingPageConfig: VenueLandingPageConfig; // @debt should this be potentially undefined, or is it guaranteed to exist everywhere?
 
+  redirectUrl?: string;
+
```
Comment (mike-lvov) [inline src/types/Venue.ts:109]: Do we really need this space here?

**Review by mike-lvov — APPROVED** [review]

File: src/pages/VenueLandingPage/VenueLandingPage.tsx [hunk 2]
```diff
@@ -53,6 +53,13 @@ export const VenueLandingPage: React.FunctionComponent<VenueLandingPageProps> =
   );
   const purchaseHistory = useSelector(userPurchaseHistorySelector);
 
+  const redirectUrl = venue?.config?.redirectUrl ?? "";
```
Comment (0xdevalias) [inline src/pages/VenueLandingPage/VenueLandingPage.tsx:56]: There is no need to set a default value with ?? here. It’s semantically more useful/correct that if it isn’t set, that the variable remains undefined. By setting it to an empty string we’re just kind of shadowing/hiding the reality of it.

in this specific case it doesn’t make a huge difference to the logic/flow of things as it’s checked right below, but this is n anti-pattern that can lead to much bigger/more annoying/harder to track down bugs, so calling it out here so people can learn and make sure not to repeat it on a winder scale.

File: src/pages/VenueLandingPage/VenueLandingPage.tsx [hunk 3]
```diff
@@ -53,6 +53,13 @@ export const VenueLandingPage: React.FunctionComponent<VenueLandingPageProps> =
...
+  const { hostname } = window.location;
+
+  if (redirectUrl && redirectUrl !== hostname) {
```
Comment (0xdevalias) [inline src/pages/VenueLandingPage/VenueLandingPage.tsx:59]: What value gets set in the DB for this? I would have expected to see code here that checks new URL(valueFromDB).hostname or similar. Have we tested that this works as expected?

Comment (0xdevalias) [reply src/pages/VenueLandingPage/VenueLandingPage.tsx:59]: And I guess if the value in the DB is just a hostname, then calling it ‘url’ seems misleading/unintuitive.

**Pull request merged**